<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>646</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>976</width>
    <height>646</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>976</width>
    <height>646</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vista Nutrizionista</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>646</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>VisualizzaNutrizionista.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="button_modifica">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>140</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>MODIFICA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_elimina">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>220</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>ELIMINA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_conferma">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>570</y>
     <width>141</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string>CONFERMA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nome">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>173</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cognome">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>218</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="codiceFiscale">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataDiNascita">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="eta">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefono">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="licenza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
color:black;
border-radius: 15px;
</string>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="VisualizzaNutrizionista.qrc"/>
 </resources>
 <connections/>
</ui>
